.travis.yml

```diff
@@ -12,6 +12,7 @@ matrix:
   fast_finish: true
 
 before_install:
+  - git rev-parse --verify HEAD
```

Comment:
Running

```
$ git rev-parse --verify HEAD
```

locally yields

```
b058d3656110b4e88a56bc14efb719d307eb292d
```

- let's see what we can see on Travis!

Reply:
Interesting, it yields

```
34ade8d616448824ade016432ed961b46ff1cd03
```

See https://travis-ci.org/joindin/joindin-api/jobs/607805091#L251-L252.

See 34ade8d616448824ade016432ed961b46ff1cd03.

Reply:
Which points here: https://github.com/joindin/joindin-api/commit/34ade8d616448824ade016432ed961b46ff1cd03

Is Travis not pulling updated branches since you're force pushing? maybe do a git fetch before?